client/src/components/landingPage/LandingPageBox.jsx

```diff
@@ -1,7 +1,56 @@
+import { motion } from "framer-motion";
```

Comment:
- I don't think we have framer-motion installed on `client` 
- npm install framer-motion into the`client` directory.

### Screenshot from the client package.json
<img width="479" alt="Screenshot 2024-02-21 at 1 14 53 AM" src="https://github.com/KyoshiNoda/FitnessHub/assets/62672803/b506dd61-b647-4d34-9c9e-8d1c9e437ad7">